### PR TITLE
Feature/normalization for mbrl

### PR DESCRIPTION
**Pull request by qiaoting159753**
Normalization of the states and prediction of state differences (next_state - curr_state) have been confirmed to help train a world model. 

This pull request: 
memory_buffer.py:
1. add a function to compute state statistics based on stored transitions. 

helpers.py:
1. add normalize and denormalize functions. (It is used in SimpleDynamic and EnsembleRewardWorld)

### CODE REVIEW

File: cares_reinforcement_learning/networks/world_models/ensemble_integrated.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import torch.utils
 from torch import optim
 import numpy as np
-from cares_reinforcement_learning.util.helpers import normalize_obs_deltas
+from cares_reinforcement_learning.util.helpers import normalize_observations_deltas
```
Comment (beardyFace) [inline cares_reinforcement_learning/networks/world_models/ensemble_integrated.py:10]: observation* remove the s

File: cares_reinforcement_learning/util/helpers.py [hunk 2]
```diff
@@ -34,6 +34,39 @@ def weight_init(m):
...
+    Normalize the states based on the statistics from experience replay.
+
+    Keyword arguments:
+        obs -- input states
+        statistics -- statistics from experience replay (no default)
+    """
+    return (obs - statistics["observation_mean"]) / statistics["observation_std"]
+
+
+def unnormalize_observations_deltas(normalized_deltas, statistics):
```
Comment (beardyFace) [inline cares_reinforcement_learning/util/helpers.py:49]: denormalize - be consistent between un and de

**Review by beardyFace — CHANGES_REQUESTED** [review]

**Review by beardyFace — APPROVED** [review]
